Project-Descriptions-and-Plans/CVA6/CVA6-main-plan-approved-2024+.md

```diff
@@ -0,0 +1,524 @@
+
+## Project Leader(s)
+
+For the main project:
+- Project Manager (PM) and Technical project leader (TPL): Jérôme Quévremont, TRT
+- Verification leader and main committer: Jean-Roch Coulon, TSS
+- FPGA softcore leader: Sébastien Jacq, TRT
+
+The sub-projects will name their leadership teams.
+
+## Project Planning Documents
+
+The main project workplan is below.
+
+## Summary of requirements
+
+### CVA6 features
+
+The CVA6 specification was prepared for the 2022-02-28 PA gate and approved by the Technical WG. It is available in [ReadTheDocs](https://docs.openhwgroup.org/projects/cva6-user-manual/02_cva6_requirements/cva6_requirements_specification.html).
+
+To avoid duplications and inconsistencies, it is not copied here.
+
+### License scheme
+
+Because of the nature of its business, Thales expectation is to contribute and get a **sustainable open-source solution**
+- to integrate CVA6 in new ASIC and FPGA projects for a long period and to maintain/upgrade them for several decades for industrial domains (avionic, satellite, railways, energy...);
+- to permit audit and reviews of verification and tools in the context of certifiable security and functional safety;
+- to permit public review to improve the quality of the core and its ecosystem;
+- to foster cooperative projects and connections with academy and research.
+
+Thereforce, Thales expects:
+- Not only the core RTL, but also a version of the verification environment and the SW tools are available as open-source.
+- Commercial tools can be added to this basic set to deliver additional value (improved quality, coverage...).
+    - But the project shall not depend on a single-source commercial tool.
+	- Commercial tools with multiple sources, such as logic simulators, are acceptable.
+
+In addition, this sustainable open-source solution, lowers barriers for newcomers to OpenHW. They can evaluate CVA6, adopt it, join OpenHW Group, add their "secret sauce"...
+
+### Verification
+
+To foster cooperation and efficiency within OpenHW Group, CVA6 will use and contribute to **core-v-verif**.
+
+**Spike** will be used as a reference ISS for some configurations.
+Synopsys/Imperas OVPSim can be considered for other configurations.
+
+Gateways between OpenHW core-v-verif repositories and Thales internal environment (GitLab, CI...) will be set up.
+
+The verification sub-projects plans will provide more insight into the verification methods.
+
+## Future enhancements (off project):
+
+Increasing the core architectural performance (out-of-order...), adding RISC-V extensions, adhering to a RISC-V profile, delivering a larger subsystem (SMP multi-core, peripherals, accelerators...)
+
+## Explanation of why OpenHW should do this project
+
+Like RI5CY, the ARIANE core donated by ETH Zürich and U. Bologna was at the heart of the OpenHW Group creation. We need an application processor core in our portfolio.
+
```

Comment:
We need to address (somewhere) that CVA6 is no longer tied to "application" (linux class) processor and the need is also for embedded variants and why CVA6 is the baseline for those embedded variants.

Reply:
Thanks. I'll update at several places.